src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -183,7 +183,15 @@ private static bool InvalidLevel(int? level)
                 case SyntaxKind.GetAccessorDeclaration:
                     {
                         var t = (AccessorDeclarationSyntax)node;
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, t.Body, cancellationToken));
+                        if (t.Body != null && t.ExpressionBody != null)
```

Comment:
Please add a test for this case if possible. Perhaps: `object P { get { return null; } => null; }`.